packages/bruno-converters/src/postman/postman-translations.js

```
@@ -4,11 +4,35 @@ import translateCode from '../utils/postman-to-bruno-translator';
 const replacements = {
-  'pm\\.environment\\.get\\(': 'bru.getEnvVar(',
-  'pm\\.environment\\.set\\(': 'bru.setEnvVar(',
-  'pm\\.variables\\.get\\(': 'bru.getVar(',
-  'pm\\.variables\\.set\\(': 'bru.setVar(',
+  // Environment variables
+  'pm\\.environment\\.get\\(': 'bru.environment.get(',
+  'pm\\.environment\\.set\\(': 'bru.environment.set(',
+  'pm\\.environment\\.has\\(': 'bru.environment.has(',
+  'pm\\.environment\\.unset\\(': 'bru.environment.unset(',
+  'pm\\.environment\\.replaceIn\\(': 'bru.interpolate(',
+  'pm\\.environment\\.toObject\\(': 'bru.environment.toObject(',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)